content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -61,26 +49,36 @@ The preceding command creates a
 
 1. Display information about the Deployment:
 
+        ```shell
```

Comment:
@RA489 , Would you remove the indentation of this code block and check the other code blocks on the page?
Thanks!

Reply:
Not sure if we want to make this part of the content style guideline. Marking code blocks using indentation is fragile. Maybe we should promote explicit code block marking?

Reply:
@kbhawkey sure i will check and remove.

Reply:
@kbhawkey removed the identation of the code.

Reply:
👀